src/structures/vroom/input/input.cpp

```diff
@@ -106,9 +106,6 @@ void Input::add_shipment(const Job& pickup, const Job& delivery) {
   if (pickup.priority != delivery.priority) {
     throw Exception(ERROR::INPUT, "Inconsistent shipment priority.");
   }
-  if (!(pickup.pickup == delivery.delivery)) {
-    throw Exception(ERROR::INPUT, "Inconsistent shipment amount.");
-  }
```

Comment:
Needs to be removed indeed to allow "asymmetric" amounts for pickup and deliveries.